roles/collectd/tasks/main.yml

```diff
@@ -1,12 +1,11 @@
 ---
-- name: install collectd
-  yum:
-      name: '{{ item }}'
-      state: present
-  with_items:
-      - collectd
```

Comment:
I understand that collectd plugins probably have stated collectd as dependency, but wouldn't it be better to have collectd installation explicit?